daffodil-runtime1/src/main/scala/org/apache/daffodil/debugger/InteractiveDebugger.scala

```diff
@@ -1497,6 +1482,17 @@ class InteractiveDebugger(runner: InteractiveDebuggerRunner, eCompilers: Express
         }
       }
 
+      object InfoHidden extends DebugCommand with DebugCommandValidateZeroArgs {
+        val name = "hidden"
+        override lazy val short = "hid"
```

Comment:
I think if you don't define ``short`` then it just defaults to the first letter of the name, which in this case is "h" and is a reasonable short form for ``hidden``. Nothing else starts with h.